docs/examples_gcp.rst

```diff
@@ -105,13 +105,13 @@ only schedules one-off jobs at present::
   #      }
   #  }
 
-The `gcp_transfer_from_aws()` method of the `encode_utils.connection.Connection` class
+The `gcp_transfer_from_aws()` method of the `igvf_utils.connection.Connection` class
 ---------------------------------------------------------------------------------------
-Requires that the user has AWS key permissions on the ENCODE buckets and file objects.
+Requires that the user has AWS key permissions on the IGVF buckets and file objects.
 
 ::
 
-  import encode_utils.connection as euc
+  import igvf_utils.connection as euc
   conn = euc.Connection("prod")
   # In production mode, the S3 source bucket is set to encode-files. In any other mode, the
   # bucket is set to encoded-files-dev.
```

Comment:
I am wondering if we should not drop the bucket specific instructions for now. we don't have files/buckets - so why do we provide documentation for functionality that would not work?

Reply:
in any case should probably not include things like "encode-files-dev" etc.